associacao-brasileira-de-normas-tecnicas-ufrgs.csl

```diff
@@ -25,15 +25,29 @@
       <name>Luísia Alves</name>
       <email>luisia.alves@bc.ufrgs.br</email>
     </author>
+    <author>
+      <name>Kelson Anthony de Menezes</name>
+      <email>kanthony@ebsco.com</email>
+      <uri>https://www.mendeley.com/profiles/kelson-anthony/</uri>
+    </author>
+    <author>
+      <name>Michelangelo Viana</name>
+      <email>mviana@ebsco.com</email>
+      <uri>
+http://www.mendeley.com/profiles/michelangelo-viana/
+</uri>
+    </author>
     <contributor>
       <name>Leticia Strehl</name>
       <uri>http://www.mendeley.com/profiles/leticia-strehl/</uri>
     </contributor>
     <category citation-format="author-date"/>
     <category field="generic-base"/>
-    <summary>The Brazilian Standard Style in accordance with ABNT-NBR 10520.2002 and ABNT-NBR 6023.2002</summary>
-    <updated>2017-10-27T14:43:40+00:00</updated>
-    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+    <summary>The Brazilian Standard Style in accordance with ABNT-NBR 10520.2002 and ABNT-NBR 6023.2018</summary>
+    <updated>2019-12-13T14:54:20+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">
+This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License
```

Comment:
Put on a single line -- this breaks the automated check

Reply:
Done. Thanks!